package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "node-non-blocking-weather-lab",
+  "version": "1.0.0",
+  "description": "Weather App Lab",
+  "main": "index.js",
+  "scripts": {
+    "test": "node_modules/mocha/bin/mocha test"
```

Comment:
Packages like `mocha` that declare a `bin` filed in their `package.json` will have those modules added to a `node_modules/.bin/[name of executable]` in case those files get moved in the source package. Mocha's been stable for a while, but just wanted to note that `node_modules/.bin/mocha` would also work and be a bit shorter :)